source/stylesheets/_tab-code_signing.scss.erb

```diff
@@ -15,106 +15,125 @@
 
 	section.certificate-help {
 		position: relative;
-		@include margin-y(30px);
-		padding: 25px $section-horizontalspacing;
+		margin: 25px $section-horizontalspacing;
+		padding: 12px 18px;
+		border: 1px solid #3bc3a3;
+		@include border-radius($def-radius);
 		background-color: $def-green-ultralight;
 
-		ul {
-			@include margin-y(0);
-			padding-left: 0;
-			list-style: none;
+		.toggleabble-support-info{
+			@include display-flex;
+			@include justify-content(space-between);
+			@include align-items(center);
 
-			li {
+			.oneliner {
 				@include display-flex;
+				@include align-items(center);
+				height: 20px;
```

Comment:
The content should specify its height. What if we change the text and it fits only in 2 lines? What if we shrink browser to a size where current text breaks to 2 lines?